Makefile

```diff
@@ -333,6 +335,9 @@ build-windows: .prebuild prepare-enterprise
 
 build: build-linux build-windows build-osx
 
+build-current: .prebuild prepare-enterprise
```

Comment:
Let's not make this new random target just for default. I don't want to maintain it. Just make the default run, that's what everyone should be doing.